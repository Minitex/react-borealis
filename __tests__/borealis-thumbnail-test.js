import React from 'react'
import renderer from 'react-test-renderer'
import BorealisThumbnail from '../src/borealis-thumbnail'
import { MemoryRouter } from 'react-router-dom';

describe('BorealisThumbnail Tests', () => {

  it("renders an audio", function(){
    const component = renderer.create(
      <MemoryRouter>
       <BorealisThumbnail src="http://example.com" thumbnail="blerg" to="foo" />
      </MemoryRouter>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
