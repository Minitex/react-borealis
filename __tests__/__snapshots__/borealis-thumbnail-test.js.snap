exports[`BorealisThumbnail Tests renders an audio 1`] = `
<div>
  <a
    className="borealis-thumbnail"
    href="/foo"
    onClick={[Function]}
    style={undefined}>
    <img
      alt="Click to load the foo"
      src="http://example.com" />
  </a>
</div>
`;
